tools/distribution/UPDATING.md

```diff
@@ -0,0 +1,157 @@
+# Updating this repository
```

Comment:
It may not be 100% applicable to Spark, but this doc describes updating the `tool` and `testing` folders in addition to the SDK version.